src/emulator.py

```diff
@@ -917,8 +917,7 @@ def get_debug_state() -> Dict[str, Any]:
                     val = val.decode("utf-8")
                 except UnicodeDecodeError:
                     val = val.hex()
-
-            debug_state_dict[key] = val
+                debug_state_dict[key] = val
```

Comment:
This does not seem right, it means we will only save byte values

Reply:
yes, as i mentioned in the description and also discussed [on slack](https://satoshilabs.slack.com/archives/CL1D61PQF/p1728639551657579), im aware that this isnt right but user-env crasches without it (tries to serialize complex field)

Reply:
I pushed a fixup commit into this branch/PR, can you please confirm it works fine for you? I can merge it in a positive case

Reply:
works for me